### PR TITLE
Create itm_encounter.json

### PR DESCRIPTION
**Pull request by drawntotheflame**
I've handtyped Into the Maelstrom. @zzorba would you be able to check against cards? Many thanks.

### CODE REVIEW

File: pack/tic/itm_encounter.json [hunk 1]
```diff
@@ -0,0 +1,578 @@
...
+        "faction_code": "mythos",
+        "flavor": "You refuse to be a pawn in the government's schemes. Even so, you don't wish Agent Harper to come to any harm. You tell her to escape while she can, and she slips away, cursing under her breath. Now you have a new goal - send the riches of Y'ha-nthlei into the depths so they can never again be used against humanity.",
+        "illustrator": "Carlos Palma Cruchaga",
+        "name": "City of the Deep (v. III)",
+        "pack_code": "itm",
+        "position": 318,
+        "quantity": 1,
+        "stage": 2,
+        "text": "<b>Objective</b> - Collapse the vault containing the Deep Ones' riches. If Dagon and Hydra each have at least 4 [per_investigator] damage on them, advance.",
+        "type_code": "act"
```
Comment (zzorba) [inline pack/tic/itm_encounter.json:153]: Should these three acts all have  `"victory": 2` on them?  That's usually how we handle victory, but I think this is the first time the back side of an act/agenda has had 'victory' on it.

File: pack/tic/itm_encounter.json [hunk 2]
```diff
@@ -0,0 +1,578 @@
...
+        "code": "07330b",
+        "encounter_code": "into_the_maelstrom",
+        "encounter_position": 25,
+        "enemy_damage": 1,
+        "enemy_evade": 4,
+        "enemy_fight": 7,
+        "enemy_horror": 2,
+        "faction_code": "mythos",
+        "health": null,
+        "hidden": true,
```
Comment (zzorba) [inline pack/tic/itm_encounter.json:409]: This is backwards I believe.

The side that contains 'back_link' is the 'front', and should not be marked hidden.  The side without back_link should be marked hidden.

In this case, I think the correct fix is to move `"back_link": "07330b"` to the a card

**Review by zzorba — APPROVED** [review]
Aside from back links on hydra/dagon and the victory stuff we discussed, changes look good!

**Comment by drawntotheflame** [comment]
@Kamalisk  this can go live now. Thanks.

**Pull request merged**